frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/setup.tsx

```diff
@@ -65,4 +69,6 @@ export const setup = ({
       initialRoute,
     },
   );
+
+  await waitFor(() => screen.getByText(/general/i));
```

Comment:
Awaiting the sidebar to load here prevents us from needing to do it manually in every other test